pystiche_papers/johnson_alahi_li_2016/_modules.py

```diff
@@ -202,6 +208,21 @@ def encoder(instance_norm: bool = True,) -> pystiche.SequentialModule:
 def decoder(
     impl_params: bool = True, instance_norm: bool = True,
 ) -> pystiche.SequentialModule:
+    r"""Decoder part of the Transformer from :cite:`JAL2016` .
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper.
+
+    If ``impl_params is True``, a scaled 150 * :func:`~torch.tanh` is used instead of
+    the (:func:`~torch.tanh`(x) + 1) / 2 in the paper. The scaling in the second variant
+    to [0, 255] is then carried out in the
+    :func:`pystiche_papers.johnson_alahi_li_2016.postprocessor`.
```

Comment:
We need to talk about that, since its not that simple. Message me when you have time to discuss this in more detail.